src/core/annotation.js

```diff
@@ -332,6 +337,30 @@ class Annotation {
       this._streams.push(this.appearance);
     }
 
+    let actions = null;
+    let fieldName = null;
+    let kidsId = null;
+    if (params.collectFields) {
+      // Fields can act as container for other fields and have
+      // some actions even if no Annotation inherit from them.
+      // Those fields can be referenced by CO (calculation order).
+      const kids = dict.getRaw("Kids");
```

Comment:
Something I didn't think of before, but are you *absolutely* sure that the /Kids entry itself cannot ever be a Reference?

Looking at the PDF specification, specifically https://www.adobe.com/content/dam/acom/en/devnet/acrobat/pdfs/PDF32000_2008.pdf#page=441, it doesn't say anything about indirect objects but there a risk that PDF documents with such data could exist.

Basically, I'd suggest using `const kids = dict.get("Kids");` instead here to prevent any possible errors. (Note that that method will *not* resolve the elements within the Array, only the Array itself.)

Reply:
Thx and fixed.